client/app/components/dialog-content/dialog-content.html

```diff
@@ -20,7 +20,7 @@ <h2 class="text-capitalize no-wrap">{{ ::vm.dialog.label }}</h2>
                 ng-repeat="dialogField in ::dialogGroup.dialog_fields "
                 ng-init="inputTitle = dialogField.description"
                 class="form-horizontal">
-                <div class="form-group">
+                <div ng-show="dialogField.visible" class="form-group">
```

Comment:
prefer ng-if to ng-show (if removes hidden elements from dom, show puts an equivalent of a display none, can sometimes interfere with elements on the page)